cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -94,7 +96,11 @@ func SetDefaults_MasterConfiguration(obj *MasterConfiguration) {
 // SetDefaults_NodeConfiguration assigns default values to a regular node
 func SetDefaults_NodeConfiguration(obj *NodeConfiguration) {
 	if obj.CACertPath == "" {
-		obj.CACertPath = DefaultCACertPath
+		if runtime.GOOS == "windows" {
```

Comment:
Please don't put `if windows { } else { }` generally in the code here. Use generic path helper funcs where there is a difference between platforms